### PR TITLE
Modified placeholder functions and text for has_transformed_position

### PR DESCRIPTION
**Pull request by kameyer226**

### CODE REVIEW

**Comment by untzag** [comment]
Thanks @kameyer226, I've been using the term "overload" wrong this whole time. Looks like override is the correct term when the method is redefined through inheritance and the function signature remains unchanged.

File: yaqd-core/yaqd_core/_has_transformed_position.py [hunk 1]
```diff
@@ -30,27 +30,29 @@ def _relative_to_transformed(self, relative_position):
...
-        Note that overloads should still obey inversion:
+        Daemons that need more than a reference_position should override this method.
+        Note that override should still obey inversion:
         ```
         _relative_to_transformed(_transformed_to_relative(x)) == x
         ```.
         """
-        return relative_position
+        transformed_position = 1.000 * relative_position
+        return transformed_position
```
[the remark this answers is not in the document]
Comment (kameyer226) [reply yaqd-core/yaqd_core/_has_transformed_position.py:40]: The text already indicates a function that essentially returns x is put in as placeholder, so I made it look more like that for pedagogical reasons.   It looks like non-maintainers may look at this section as a blueprint.

[the remark this answers is not in the document]
Comment (untzag) [reply yaqd-core/yaqd_core/_has_transformed_position.py:40]: OK, I see your motivation now. I disagree, but I'm not too interested in getting into a debate about code style. Willing to merge.

**Review by untzag — CHANGES_REQUESTED** [review]
@kameyer226 please edit CHANGELOG and I will merge

**Comment by ksunden** [comment]
I am actually quite strongly against the "multiply by 1.000" here...

I do not think it actually is pedagogically helpful

- it is for one thing implying something of the precision by having 3 zeros
- it is doing extra computation for no run-time reason (and many of the usecases of this code are simply to have an offset, so these are often left as is)

**Comment by untzag** [comment]
Since there is some debate, I'm going to hop on and remove the multiplication / reassigning variable names piece from this PR.

@kameyer226 you can reopen the multiplication code in a separate PR if you wish to, we can discuss it there on its own merits.

As a thought, perhaps you could contribute some examples to the YEP instead of changing the code? That might be a more productive way to give new users clues about how to use this trait.

I'll merge the docs changes now. Thanks for the PR.

**Review by untzag — APPROVED** [review]

**Pull request merged**